package.json

```diff
@@ -19,6 +19,7 @@
   ],
   "license": "GPL-2.0",
   "devDependencies": {
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
https://babeljs.io/docs/plugins/preset-env/

Preferable if you use this babel module over the 3s2015 preset.

Reply:
nice, thanks